app/templates/sites/new.hbs

```diff
@@ -29,11 +32,7 @@
       </:action>
     </PageHeader>
 
-    <form
-      id="new-vestiging-form"
-      {{on "submit" (perform this.saveTask)}}
```

Comment:
Make a seperate action for this, as suggested in my other comment. Make an action
`@action handleAddClick(event)`. In this action validate and then call saveTask.perform.